app/assets/javascripts/Results/context_menu.js.erb

```diff
@@ -16,35 +16,39 @@ var annotation_context_menu = {
   setup: function(annot_path, result_id, assignment_id, file_dl_path) {
     var menu_items = {
       new_annotation: {
-        title: 'New Annotation',
+        title: '<%= I18n.t('marker.annotation.context_menu.new') %>',
```

Comment:
Ah yes, you're right. I'll make a note of this, but you don't have to change these for now.